docs/Installation/InstallationOnClusters.md

```diff
@@ -20,9 +20,10 @@ want to load the modules, source the shell file for your system and run
    `SYSTEM_TO_RUN_ON` is replaced by the name of the system as described in the
    relevant section below.
 5. If you haven't already, choose where you want to install the dependencies,
-   e.g. into `SPECTRE_DEPS` and run `spectre_setup_modules SPECTRE_DEPS`. This
+   e.g. into `SPECTRE_DEPS` and run `spectre_setup_modules $SPECTRE_DEPS`. This
    will take a while to finish. Near the end the command will tell you how to
-   load make the modules available by providing a `module use` command.
+   load make the modules available by providing a `module use` command. Make
+   sure you are providing an absolute path to `spectre_setup_modules`.
```

Comment:
Maybe we should change the script so it works with a relative path.

Reply:
Should that be addressed in this PR or should it be made an issue?

Reply:
I think a sufficient change to the script is to add ``dep_dir=`pwd` `` after the first `cd $dep_dir`, but since that will require some testing you can just make an issue if you'd like.  (Also, `dep_dir` should be declared local when it is first used.)

Reply:
@gsb76 can you make an issue, and then we can merge this